charts/sysdig-deploy/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A chart with various Sysdig components for Kubernetes
 
 type: application
 
-version: 1.3.11
+version: 1.3.12
 
 maintainers:
   - name: achandras
```

Comment:
you'd also need to bump the agent chart version on line 23 in order to include your change in that sub-chart